cost/aws/reserved_instances/recommendations/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v2.18
+
+- Renamed `Reserved Instance` policy_set from info block to `Reserved Instances`.
```

Comment:
Should be reworded to be more sensible to users that don't know about the policy template language and won't know what an "info block" is. For example:

- Changed policy set from `Reserved Instance` to `Reserved Instances` to match other similar policies.